packages/db/src/lib/granule.ts

```diff
@@ -19,8 +19,8 @@ import { UpdatedAtRange } from '../types/record';
 const { TableNames } = require('../tables');
 
 export const getGranuleCollectionId = async (
-  knexOrTransaction: Knex | Knex.Transaction,
-  granule: PostgresGranule | PostgresGranuleRecord
+  knexOrTransaction: Knex,
```

Comment:
So, I grok restricting the typing, in that our callers currently aren't using a transaction, however most of the db/lib functions are intentionally inclusive of transactions.         This is likely a user-facing breaking change,   we should chat briefly about it.

Reply:
The `Transaction` interface extends the `Knex` interface, so any `Knex.Transaction` will meet this requirement.

Reply:
Or, said another way, if a user passes a `Knex.Transaction` in here, it'll work just fine because a `Transaction` is a valid `Knex`.

Reply:
Ah - so it does.  I'm wondering if we missed that in an upgrade typings-wise, or if it was just a mistake.